test/image.bats

```diff
@@ -516,3 +516,32 @@ EOF
 
 	run crictl run "$TESTDIR"/memory.json "$TESTDATA"/sandbox_config.json
 }
+
+@test "run container having memory limit below default 12 MiB with empty container_min_memory" {
+	cat << EOF > "$CRIO_CONFIG_DIR/99-mem.conf"
+[crio.runtime]
+default_runtime = "mem"
+[crio.runtime.runtimes.mem]
+runtime_path = "$RUNTIME_BINARY_PATH"
+EOF
+	start_crio
+
+	case $(go env GOARCH) in
+	amd64)
+		crictl pull ${IMAGE_LIST_DIGEST_FOR_TAG_AMD64}
+		IMAGE=${IMAGE_LIST_DIGEST_FOR_TAG_AMD64}
+		;;
+	arm64)
+		crictl pull ${IMAGE_LIST_DIGEST_FOR_TAG_ARM64}
+		IMAGE=${IMAGE_LIST_DIGEST_FOR_TAG_ARM64}
+		;;
+	esac
+
+	jq --arg image "$IMAGE" '.metadata.name = "memory"
+		| .command = ["/bin/sh", "-c", "sleep 600"]
+    | .linux.resources.memory_limit_in_bytes = 12582911
+		| .image.image = $image' \
+		"$TESTDATA"/container_config.json > "$TESTDIR"/memory.json
+
+	run ! crictl run "$TESTDIR"/memory.json "$TESTDATA"/sandbox_config.json
```

Comment:
`run` here won't report whether the command failed or not, it'll save it in variables. I recommend dropping `run` if we expect this command to fail

Reply:
according to the doc https://bats-core.readthedocs.io/en/stable/writing-tests.html#when-not-to-use-run,

> Note: In contrast to the above, testing that a command failed is best done via
`run ! command args ...`

However I think just checking the log line is enough as @/roman-kiselenko said.

I'm going to remove this test and add log check to the existing test.